Project.toml

```diff
@@ -7,14 +7,15 @@ version = "0.5.0"
 CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
+Downloads = "f43a241f-c20a-4ad4-852c-f6b1247861c6"
 Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
 NearestNeighbors = "b8a86587-4115-5ab1-83bc-aa920d37bbce"
 ZipFile = "a5390f91-8eb1-5f08-bee0-b1d1ffed6cea"
 
 [compat]
-julia = "1"
 CSV = "0.10"
 DataFrames = "1"
 DataStructures = "0.18"
 NearestNeighbors = "0.4"
 ZipFile = "0.10"
+julia = "1"
```

Comment:
We need a compat for Downloads for (the package to be automatically registered)[https://juliaregistries.github.io/RegistryCI.jl/stable/guidelines/#New-versions-of-existing-packages], so `Downloads =  "1.6"` should do the trick.